front/src/modules/ui/modal/components/ConfirmationModal.tsx

```diff
@@ -25,14 +25,19 @@ export type ConfirmationModalProps = {
 };
 
 const StyledConfirmationModal = styled(Modal)`
-  padding: ${({ theme }) => theme.spacing(4)};
-  width: calc(400px - ${({ theme }) => theme.spacing(10 * 2)});
+  border-radius: ${({ theme }) => theme.spacing(1)};
+  padding: ${({ theme }) => theme.spacing(6)};
+  width: calc(400px - ${({ theme }) => theme.spacing(32)});
```

Comment:
I wonder if won't be better to pass a props to the Modal instead of overriding its style.
Bot work as we are only touchy the Modal outer container so I'm fine with it.

Reply:
Where does the theme.spacing(32) come from? Looks weird to me!

Reply:
Agree.
I changed a 20 to 32 to be aligned with the figma